lib/topological_inventory.rb

```diff
@@ -0,0 +1,10 @@
+require 'topological_inventory-api-client'
+class TopologicalInventory
+  def self.api
+    TopologicalInventoryApiClient.configure do |config|
+      config.host     = ENV['TOPOLOGICAL_INVENTORY_URL']
+      config.scheme   = URI.parse(ENV['TOPOLOGICAL_INVENTORY_URL']).try(:scheme)
+    end
+    TopologicalInventoryApiClient::DefaultApi.new
```

Comment:
Does it makes sense to memoize this?  We probably shouldn't create a brand new client object on every single call.

Reply:
@Fryguy This is just setting up an api_instance with the additional headers for x-rh-auth-identity based on the current user making the request should we share it. It also brings in Thread safety when multiple puma threads would be using the shared api instance

Reply:
That's not what I'm saying though...this will literally create a DefaultApi instance each time.